react-client-app/src/components/ScanReportTbl.jsx

```diff
@@ -32,37 +32,50 @@ const ScanReportTbl = (props) => {
         scanreports = scanreports.sort((b, a) => (a.id > b.id) ? 1 : ((b.id > a.id) ? -1 : 0))
         // for each scan report use the data partner and author ids to get name to display
         // get list of unique data partner and auther ids
-        const dataPartnerObject = {}
-        const authorObject = {}
-        scanreports.map(scanreport => {
-            dataPartnerObject[scanreport.data_partner] = true
-            authorObject[scanreport.author] = true
-            const created_at = {}
-            created_at.created_at = scanreport.created_at
-            created_at.displayString = moment(scanreport.created_at.toString()).format('MMM. DD, YYYY, h:mm a')
-            scanreport.created_at = created_at
-        })
-        // make batch queries with author ids and data partner ids
-        const dataPartnerIds = chunkIds(Object.keys(dataPartnerObject))
-        const authorIds = chunkIds(Object.keys(authorObject))
-        const dataPartnerPromises = []
-        const authorPromises = []
-        for (let i = 0; i < dataPartnerIds.length; i++) {
-            dataPartnerPromises.push(useGet(`/datapartners/?id__in=${dataPartnerIds[i].join()}`))
-        }
+        const authorObject = {};
+        const datasetObject = {};
+        scanreports.map((scanreport) => {
+            authorObject[scanreport.author] = true;
+            datasetObject[scanreport.parent_dataset] = true;
+            const created_at = {};
+            created_at.created_at = scanreport.created_at;
+            created_at.displayString = moment(scanreport.created_at.toString()).format("MMM. DD, YYYY, h:mm a");
```

Comment:
MMM. ? seems odd format?

Reply:
MMM means the month shortened to 3 letters e.g, Jan, Feb, etc